infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -265,7 +268,8 @@ public TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
                     if (v > txSnapshotTimestamp) {
                         log.debug("ABORT[{}] conflict-stream[{}](ts={})",
```

Comment:
Again no harm logging this as warn or info. it might be useful debug info. you can change the level once the system is more stable.